hazelcast/src/main/java/com/hazelcast/sql/impl/partitioner/AllFieldsRowPartitioner.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.sql.impl.partitioner;
+
+import com.hazelcast.internal.serialization.InternalSerializationService;
+import com.hazelcast.nio.ObjectDataInput;
+import com.hazelcast.nio.ObjectDataOutput;
+import com.hazelcast.sql.impl.row.Row;
+
+import java.io.IOException;
+
+/**
+ * Hash function which uses all row columns to calculate the hash.
+ */
+public class AllFieldsRowPartitioner extends AbstractFieldsRowPartitioner {
```

Comment:
Why do we need `AllFieldRowPartitioner` for sorting that always sends data to a single member? Perhaps this class should not be part of the PR.

Reply:
Removed as part of the https://github.com/hazelcast/hazelcast/pull/18013/commits/ba2cf9f28770bf86017f73cd451ee28e07125f19